language-design.md

```diff
@@ -62,7 +175,50 @@ the commenters?
 
 **Response**
 
-
+Pavlus is arguing that programming languages should be designed to be easier
+for the average person to understand and learn. While we agree with this
+general argument of the article, the actual specifics Pavlus uses to bolster
+his claim are not convincing. The commenters point out that Pavlus is
+emphasizing the difficulty of learning a language for those new to the
+language, but they argue that a programming language should be designed more
+with its users in mind, in that ease of use should be the concern, not ease
+of learning.
```

Comment:
Yeah, I like that distinction. But aren't ease of use and ease of learning pretty strongly related?